interface/patient_file/encounter/find_code_dynamic.php

```diff
@@ -45,7 +45,7 @@
 <head>
 <title><?php echo xlt('Code Finder'); ?></title>
 
-<?php Header::setupHeader(['opener', 'datatables', 'datatables-dt', 'datatables-colreorder']); ?>
+<?php Header::setupHeader(['opener', 'datatables', 'datatables-bs', 'datatables-colreorder']); ?>
```

Comment:
Something I've noted in other places in the codebase when bringing in bs into datatables, is that still need to keep the `datatables-dt` there.
```
<?php Header::setupHeader(['opener', 'datatables', 'datatables-dt', 'datatables-bs', 'datatables-colreorder']); ?>
```

Reply:
OK, Updated the commit.